### ISSUE TITLE
Devnet v3 todos

### ISSUE THREAD

**Issue by roberto-bayardo** [Inphi/eip4844-interop#43]
Ready additional client implementations:
CL: Lodestar, Lighthouse
EL: Erigon, Nethermind

Incorporate fastssz bug fix into Prysm **[done]**

Prysm rebase on capella (in progress)

Implement recent spec updates that affect our implementations **[done, though more updates are incoming]**

**Comment by roberto-bayardo** [comment]
[deleted - obsolete]

**Comment by roberto-bayardo** [comment]
Tim's devnet-3 spec update tracker here: https://notes.ethereum.org/@timbeiko/4844-devnet-3

**Comment by jimmygchen** [comment]
Hi @roberto-bayardo 
I'm seeing recent developments for Prysm happening on the [prysmaticlabs/prysm eip4844](https://github.com/prysmaticlabs/prysm/tree/eip4844) branch, is the plan to switch over to this repo when it's ready for devnet v3?

**Comment by roberto-bayardo** [comment]
Yeah would be best if we could align on the same repo to avoid duplicative work but I've been unsuccessful so far trying to converge these work streams. @Inphi what do you think?

**Comment by Inphi** [comment]
Agreed. I'll check with terencechain on the status of his branch.

**Comment by roberto-bayardo** [comment]
FYI this is a client breaking change: we are changing kzg trusted setup to the consensus spec version.

https://github.com/mdehoog/go-ethereum/pull/53

**Comment by roberto-bayardo** [comment]
BTW we are also moving back to using eip-4844 branches in both prysm & geth for development. I will be deleting the devnet-v3 branches.   For eip4844-interop repo we'll be using master.
